pages/index.js

```diff
@@ -26,16 +18,10 @@ export const getStaticProps = async () => {
 export default function Home({ image }) {
   return (
     <PrimaryLayout image={image} inNeed main>
-      <HotMealDaySection />
```

Comment:
Why did you take this out again??

Reply:
Sorry, I miss read it by <HotMealDayCard/>